ARCL/Source/SceneLocationViewEstimateDelegate.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  SceneLocationViewDelegate.swift
+//  ARKit+CoreLocation
+//
+//  Created by Ilya Seliverstov on 09/08/2017.
+//  Copyright © 2017 Project Dent. All rights reserved.
+//
+
+import Foundation
+import ARKit
+import CoreLocation
+import MapKit
+
+@available(iOS 11.0, *)
+public protocol SceneLocationViewEstimateDelegate: class {
+    func didAddSceneLocationEstimate(sceneLocationView: SceneLocationView, position: SCNVector3, location: CLLocation)
+    func didRemoveSceneLocationEstimate(sceneLocationView: SceneLocationView, position: SCNVector3, location: CLLocation)
+}
+
+@available(iOS 11.0, *)
+public extension SceneLocationViewEstimateDelegate {
+    func didAddSceneLocationEstimate(sceneLocationView: SceneLocationView, position: SCNVector3, location: CLLocation) {
+        //
```

Comment:
Does this need to exist?

Reply:
The idea is that they are default implementations of the functions (essentially making them optional when conforming to the protocol).  They don't have to exist, but by existing, they allow you to choose which functions you want to implement and not have no-op implementations for the ones you don't want to implement.